R/BLOBFISH.R

```diff
@@ -0,0 +1,645 @@
+library(fgsea)
```

Comment:
Hi @taraeicher , this was causing the tests to fail. You would need to import the functions you you need from fgsea but you can't call it in a function header like that.

Reply:
Hi @marouenbg, yes, I saw that. I recall that you had asked me to add this, but I must have taken the comment out of context. It can safely be removed because I don't use fgsea here. I'm verifying that the package runs locally now.

Reply:
Yes, I asked you to add it in the analysis code (supplementary material code) not in Blobfish.

Reply:
@taraeicher Supplementary_File_2_v3.R